Description: Disables loading the previous way media was sent, so it always selects either
photo or album by default.

diff --git a/Telegram/SourceFiles/boxes/send_files_box.cpp b/Telegram/SourceFiles/boxes/send_files_box.cpp
--- a/Telegram/SourceFiles/boxes/send_files_box.cpp
+++ b/Telegram/SourceFiles/boxes/send_files_box.cpp
@@ -1947,14 +1947,6 @@
 				? SendFilesWay::Album
 				: SendFilesWay::Photos;
 		}
-		const auto way = Core::App().settings().sendFilesWay();
-		if (way == SendFilesWay::Files) {
-			return way;
-		} else if (way == SendFilesWay::Album) {
-			return _list.albumIsPossible
-				? SendFilesWay::Album
-				: SendFilesWay::Photos;
-		}
-		return (_list.albumIsPossible && !_albumPhotosCount)
+		return _list.albumIsPossible
 			? SendFilesWay::Album
 			: SendFilesWay::Photos;
 	}();
